### PR TITLE
Add a demo: PlatformerWithMoment

### PR DESCRIPTION
**Pull request by buckle2000**
#223

### CODE REVIEW

**Comment by Gama11** [comment]
Can you adjust the code style to match Flixel's? Putting `{`s in a new line in particular.

**Comment by Gama11** [comment]
Also, indent seems to be a mix of tabs and spaces, Flixel uses tabs.

File: Platformers/PlatformerWithMoment/Project.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	
+	<!--------------------------------PATHS SETTINGS-------------------------------->
+	
+	<set name="BUILD_DIR" value="export" />
+	<classpath name="source" />
+	<assets path="assets" />
+	
+	<!--------------------------------LIBRARIES------------------------------------->
+	<haxelib name="openfl"/>
+	<haxelib name="actuate"/>
```
Comment (Gama11) [inline Platformers/PlatformerWithMoment/Project.xml]: Can you remove the actuate dependency? I don't want an additional dependency on Travis. Doesn't seem to be used anywhere anyway?

File: Platformers/PlatformerWithMoment/Project.xml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	<window if="mobile" orientation="landscape" fullscreen="true" width="0" height="0" />
+	
+	<!--------------------------------PATHS SETTINGS-------------------------------->
+	
+	<set name="BUILD_DIR" value="export" />
+	<classpath name="source" />
+	<assets path="assets" />
+	
+	<!--------------------------------LIBRARIES------------------------------------->
+	<haxelib name="openfl"/>
```
Comment (Gama11) [inline Platformers/PlatformerWithMoment/Project.xml]: The openfl include here should be removed, this already happens in Flixel's `include.xml`.

**Comment by buckle2000** [comment]
Done

**Comment by Gama11** [comment]
Thanks!

**Comment by Gama11** [comment]
I decided to replace the old FlxNapeTilemap with this one.

**Comment by buckle2000** [comment]
Great!

**Pull request merged**